iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -259,6 +263,23 @@ TEST_F(PoshRuntime_test, GetMiddlewarePublisherIsSuccessful)
     EXPECT_EQ(publisherOptions.historyCapacity, publisherPort->m_chunkSenderData.m_historyCapacity);
 }
 
+TEST_F(PoshRuntime_test, GetMiddlewarePublisherMaxCapacity)
```

Comment:
Very nice AAA pattern :D ...

Reply:
I am starting to understand how to write test correctly ^^